docs/user/reference/hook.md

```diff
@@ -283,14 +285,26 @@ information or advice before signalling the error.
 
 This section describes the workflows and associated hook events which are important to the operation of Juju.
 
-(action-hooks)=
-### Action hooks
+(actions)=
+### Actions
 
-Action hooks operate in an environment with additional environment variables available:
+Actions operate in an execution environment similar to a hook, with additional environment variables available:
 
 * JUJU_ACTION_NAME holds the name of the action.
 * JUJU_ACTION_UUID holds the UUID of the action.
 
+An action is used to perform a named, parameterised operation and report back the results of said operation.
+Actions are defined by the charm and invoked by the user using {ref}`command-juju-run`. The default behaviour is that
+the command blocks, logs any progress messages as reported by the action, and finally prints the action result.
```

Comment:
I might use semi-colons, just because otherwise I'm tempted to parse "blocks" as a noun:

```suggestion
the command blocks; logs any progress messages as reported by the action; and finally prints the action result.
```

Reply:
It really needs the commas rather than semicolons. But I reworded instead.